src/style.css

```diff
@@ -0,0 +1,174 @@
+.heading {
+  text-align: center;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+}
+
+.box {
+  width: 40%;
+  box-shadow: 1px 1px 2px 2px #aeabab;
+  background-color: #fff;
+}
+
+.head {
+  display: flex;
+  border-bottom: 1px solid #ddd;
+}
+
+.form-head {
+  border-bottom: 1px solid #ddd;
+  height: 50px;
+}
+
+.form {
+  display: flex;
+  height: 100%;
+}
+
+.head p {
+  flex: 9;
+  border: 0;
+  padding-left: 2%;
+}
+
+.addList {
+  flex: 9;
+  border: 0;
+  padding-left: 2%;
+  font-style: italic;
+}
+
+.addList:focus {
+  border-color: transparent; /* Remove border on focus */
+  outline: none;
+}
+
+.click-icon,
+.recycle_icon {
+  flex: 1;
+  margin-top: auto;
+  margin-bottom: auto;
+  color: #ddd;
+  display: flex !important;
+  justify-content: center;
+  cursor: pointer;
+}
+
+.checkbox-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.checkbox-list li {
+  border-bottom: 1px solid #ddd;
+}
+
+.checkbox-list input[type="checkbox"] {
+  display: none;
+}
+
+.list-item {
+  display: flex;
+}
+
+.list-text {
+  flex: 8.5;
+  border: 0;
+  height: 100%;
+  padding: 3%;
+  padding-top: 3%;
+  padding-bottom: 3%;
+  font-size: 16px;
+  color: #000;
+}
+
+.list-text:focus {
+  border-color: transparent; /* Remove border on focus */
+  outline: none; /* Remove outline on focus */
+}
+
+.checkbox-list label {
+  display: inline-block;
+  cursor: pointer;
+  flex: 0.5;
+  margin-top: auto;
+  margin-bottom: auto;
+  padding-left: 2%;
+}
+
+.checkbox-list label::before {
+  content: "";
+  display: inline-block;
+  width: 16px;
+  height: 16px;
+  border: 1px solid #ccc;
+  background-color: #fff;
+  margin-right: 5px;
+  vertical-align: middle;
+}
+
+.list-icon {
+  flex: 0.5;
+  margin-top: auto;
+  margin-bottom: auto;
+  cursor: all-scroll;
+  color: #ddd;
+}
+
+.checkbox-list input[type="checkbox"]:checked + label::before {
+  background-color: #00a0e9;
+}
+
+.checkbox-list input[type="checkbox"]:checked + label::after {
+  content: "\2713"; /* Unicode checkmark symbol */
+
+  /* display: inline-block; */
+  color: #fff;
+  font-size: 12px;
+
+  /* line-height: 16px; */
+
+  /* vertical-align: middle; */
```

Comment:
Kindly take away every commented piece of code as they make your code base look a bit messy, giving it a less professional look.😉.